script_for_using_sts_assume_role.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# For use this script you need to install jq "sudo apt install jq"
+read -p 'Input token code: ' token_key
+read -p 'Input profile name: ' profile
+json=$(aws sts assume-role --role-arn arn:aws:iam::546240550610:role/Admins \
```

Comment:
can we get role arn from profile something like 
`aws configure  --profile dos14 get role_arn`

Reply:
It doesn't work for me via aws configure --profile $profile get role_arn ((

Reply:
Ok can we somehow get the ID of profiles project?

Reply:
Create a new version script.